### PR TITLE
add xcharts v0.3.0 with npm autoupdate config

### LINKED ISSUE #7652: [Request] Add xCharts
**Library name:** xCharts
**Git repository url:** https://github.com/tenXer/xcharts/
**npm package url(optional):** https://www.npmjs.com/package/xcharts
**License(s):** MIT
**Official homepage:** http://tenxer.github.io/xcharts/
**Wanna say something? Leave message here:**

=====================
Notes from cdnjs maintainer:
Please read the README.md and CONTRIBUTING.md document first.

You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by pvnr0082t**
PR for #7652 
@x09326 Please help me review this pull request, thank you.

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/tenXer/xcharts/
 * Official website (optional, not the repository): http://tenxer.github.io/xcharts/
 * NPM package url (optional): https://www.npmjs.com/package/xcharts
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 1765
   - Count of watchers: 105
   - Count of forks: 191
 * NPM download stats (optional):
   - Downloads in the last day: 3
   - Downloads in the last week: 34
   - Downloads in the last month: 139

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #7652, cc @tenXer

### CODE REVIEW

**Comment by x09326** [comment]
@PeterDaveHello I think this PR is okay, please help me reivew it, thanks.

File: ajax/libs/xcharts/package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "xcharts",
+  "filename": "xcharts.min.js",
+  "version": "0.3.0",
+  "description": "A D3-based library for building custom charts and graphs.",
+  "homepage": "http://tenxer.github.io/xcharts/",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/tenxer/xcharts.git"
```
Comment (PeterDaveHello) [inline ajax/libs/xcharts/package.json]: ten**X**er

**Comment by pvnr0082t** [comment]
@x09326 I update ```url``` in repository field. Please help me review again, thank you.

File: ajax/libs/xcharts/package.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "xcharts",
+  "filename": "xcharts.min.js",
+  "version": "0.3.0",
+  "description": "A D3-based library for building custom charts and graphs.",
+  "homepage": "http://tenxer.github.io/xcharts/",
```
Comment (PeterDaveHello) [inline ajax/libs/xcharts/package.json]: Please try to use https for github.io domain next time, most of them prefer https.

**Comment by pvnr0082t** [comment]
@x09326 I have used https instead of http. Please help me review again, thank you.

**Comment by x09326** [comment]
@PeterDaveHello I think this PR is okay, please help me review again, thanks.

**Pull request merged**